src/engine/websites/BlogTruyen_e2e.ts

```diff
@@ -0,0 +1,25 @@
+﻿import { TestFixture, type Config } from '../../../test/WebsitesFixture';
+
+const config: Config = {
+    plugin: {
+        id: 'blogtruyen',
+        title: 'BlogTruyen'
+    },
+    container: {
+        url: 'https://blogtruyen.vn/15325/arcadia-of-the-moonlight',
+        id: '/15325/arcadia-of-the-moonlight',
+        title: 'Arcadia Of The Moonlight'
+    },
+    child: {
+        id: '/c247578/arcadia-of-the-moonlight-chap-1',
+        title: 'chap 1'
+    },
+    entry: {
+        index: 0,

```

Comment:
[This image](https://blogtruyen.vn/donate.png) looks very generic, probably better to use an image that only belongs to this chapter under test e.g., image 5.
The first images seems to be bloat, maybe these can be removed.

Reply:
done